src/components/EquipmentExpansionList.tsx

```diff
@@ -31,20 +31,25 @@ const EquipmentExpansionList: FunctionComponent<EquipmentExpansionListProps> = (
   };
 
   // Expand panel when the currently selected category is this EquipmentExpansionList's cateogory (parentId)
-  const expanded = currentCategory === equipmentList[0].category ? true : false;
+  const expanded = currentCategory === (equipmentList[0].category.path || equipmentList[0].category.name)  ? true : false;
   return (
+    <div
+      style={{
+        textTransform: "capitalize"
+      }}
+    >
     <ExpansionPanel expanded={expanded}>
       <ExpansionPanelSummary
         expandIcon={<ExpandMoreIcon />}
-        aria-label={equipmentList[0].category + "category expansion"}
-        aria-controls={equipmentList[0].category + "category expansion"}
-        id={equipmentList[0].category + "category expansion"}
+        aria-label={equipmentList[0].category.path || equipmentList[0].category.name + "category expansion"}
```

Comment:
I'm seeing this `path || name` all over the place; it's very noisy.  Can we do this once then give the result a meaningful name?

Reply:
i'm trying to get it work in a getter since that would be the best place, but having trouble with typescript getters

Reply:
Maybe using `get` isn't the right approach.  Could also just define the `category` prop differently.